components/02-elements/dropdown-list/_dropdown-list.scss

```diff
@@ -1,14 +1,32 @@
+$dropdown-list-padding              : 10px !default;
+$dropdown-list-background           : $white !default;
+$dropdown-list-font-size            : $font-size-extra-small !default;
+$dropdown-list-breakpoint-small     : $screen-m !default;
+$dropdown-list-breakpoint-medium    : $screen-l !default;
+$dropdown-list-breakpoint-large     : $screen-xl !default;
+$dropdown-list-icon-transition      : transform 0.3s, opacity 0.3s !default;
+$dropdown-list-icon-color           : $gray !default;
+$dropdown-list-icon-width           : 10px !default;
+$dropdown-list-icon-height          : 6px !default;
+$dropdown-list-icon-top             : 20px !default;
+$dropdown-list-icon-right           : 15px !default;
+$dropdown-list-icon-inner-base      : 8px !default;
+$dropdown-list-link-color           : $font-color-base !default;
+$dropdown-list-link-after-background: $white !default;
+$dropdown-list-inner-list-background: $white !default;
```

Comment:
it's really necessary? maybe just background? Please do not multiply the variable

Reply:
changed and checked in other files